test-e2e/manager-basic.js

```diff
@@ -56,19 +58,86 @@ test('Managing created projects', async (t) => {
   t.ok(project1)
   t.ok(project2)
 
-  t.test('initial settings from project instances', async (st) => {
-    const settings1 = await project1.$getProjectSettings()
-    const settings2 = await project2.$getProjectSettings()
+  t.test(
+    'initial settings and default config from project instances',
+    async (st) => {
+      const settings1 = await project1.$getProjectSettings()
+      const settings2 = await project2.$getProjectSettings()
+
+      const {
+        presets: defaultPresets,
+        fields: defaultFields,
+        icons: defaultIcons,
+      } = await getExpectedConfig(
+        new URL('../config/defaultConfig.mapeoconfig', import.meta.url).pathname
+      )
+
+      st.is(settings1.name, undefined, 'undefined name for project1')
+      st.is(settings2.name, 'project 2', 'matched name for project2')
+
+      const projectPresets = await project1.preset.getMany()
+
+      st.is(
+        settings1.defaultPresets?.point.length,
+        defaultPresets.length,
+        'the default presets loaded is equal to the number of presets in the default config'
+      )
+
+      st.alike(
```

Comment:
this asssertion - and the following in this group - are failing. From what I can seem it seems that calling `project1.preset.getMany()` returns the presets corresponding to the config that I load later, which is weird.

Reply:
It seems that is related to assertions running in an unexpected order